.github/workflows/test.yml

```diff
@@ -15,19 +15,10 @@ jobs:
         python-version: [ 3.8, 3.9, 3.11, 3.12 ]
     steps:
       - uses: actions/checkout@v2
-      - name: Setup Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v2
-        with:
-          python-version: ${{ matrix.python-version}}
-      - name: Setup
-        run: make setup
-      - name: Lint
-        run: .venv/bin/pre-commit run pylint -v
-      - name: Style
-        run: .venv/bin/pre-commit run black -v
-      - name: Typing
-        run: .venv/bin/pre-commit run mypy -v
-      - name: Wheel
-        run: make wheel
-      - name: Test
-        run: make test
+      - uses: astral-sh/setup-uv@v3
+      - run: uv run -p ${{ matrix.python-version}} black pyfu_usb/ tests/
+      - run: uv run -p ${{ matrix.python-version}} mypy pyfu_usb/ tests/
+      - run: uv run -p ${{ matrix.python-version}} pylint pyfu_usb/ tests/
+      - run: uv run -p ${{ matrix.python-version}} isort pyfu_usb/ tests/
+      - run: uv run -p ${{ matrix.python-version}} pytest tests/
+      - run: uv build -p ${{ matrix.python-version }}
```

Comment:
nit/suggestion: probably can use `ruff` instead of `isort` and `black`, too

Reply:
Good call, probably can replace `pylint` as well. Will make that change.